Cylenium.Tests/ElementTests.cs

```diff
@@ -140,6 +140,19 @@ public void Check_a_checkbox()
             Assert.That(checkbox.IsChecked(), Is.True);
         }
 
+        [Test]
+        [Category("element")]
+        public void Uncheck_a_checkbox()
+        {
+            cy.Visit("http://the-internet.herokuapp.com/checkboxes");
+
+            var checkbox = cy.Get("[type='checkbox']");
+            Assert.That(checkbox.IsChecked(), Is.False);
+
+            checkbox.Uncheck();
+            Assert.That(checkbox.IsChecked(), Is.False);
```

Comment:
I think you want to check the box and then uncheck it to make sure it unchecks correctly

Reply:
i feel that's what i tried to do initially, but the test didn't work as expected. so i changed it to this. i'll go back and give that an update though to see.